ops/github_telemetry/projects.py

```
@@ -0,0 +1,130 @@
+        if not isinstance(n, dict):
+            continue
+        items = n.get("items") or {}
+        item_nodes = list(items.get("nodes") or []) if isinstance(items, dict) else []
+        projects.append(
+            {
+                "id": n.get("id"),
+                "title": n.get("title"),
+                "number": n.get("number"),
+                "url": n.get("url"),
+                "closed": n.get("closed"),
+                "item_count": items.get("totalCount") if isinstance(items, dict) else len(item_nodes),
```

Pylint (4.0.7)

[convention] Line too long (102/100)

(C0301)